### ISSUE TITLE
Rule Change: Allow for configurable at rules

### ISSUE THREAD

**Issue by bradgarropy** [eslint/css#35]
### What rule do you want to change?

`no-invalid-at-rules`

### What change do you want to make?

Generate fewer warnings

### How do you think the change should be implemented?

A new option

### Example code

```css
@tailwind base;
@tailwind components;
@tailwind utilities;
```

### What does the rule currently do for this code?

Here is a minimal reproduction of the current state of things.
https://stackblitz.com/edit/no-irregular-whitespace-ey1hdv5b

Currently, the rule shows an error, as it does not understand Tailwind's custom `@tailwind` at rule.

```text
/home/projects/better-tailwind-support/tailwind.css
   1:1   error  Unknown at-rule '@tailwind' found                                                                           css/no-invalid-at-rules
   2:1   error  Unknown at-rule '@tailwind' found                                                                           css/no-invalid-at-rules
   3:1   error  Unknown at-rule '@tailwind' found                                                                           css/no-invalid-at-rules
```

### What will the rule do after it's changed?

We should introduce a new option to the rule, so we can add other supported custom at rules, like `@tailwind`.

I would expect the configuration to look something like this:

```javascript
"css/no-invalid-at-rules": ["error", "@tailwind", "@other", ...]
```

### Participation

- [ ] I am willing to submit a pull request to implement this change.

### Additional comments

/cc @JoshuaKGoldberg
